perf/perf_nmem.py

```diff
@@ -173,10 +173,10 @@ def test_mixed_events(self):
             mix_events = []
             for keys in self.all_events.keys():
                 mix_events.append(self.all_events[keys][0])
-            op = process.system_output("perf stat -e '{%s}' sleep 1"
-                                       % (",".join(mix_events)), shell=True,
-                                       ignore_status=True)
-            er_ln = "The events in group usually have to be from the same PMU"
+            op = process.run("perf stat -e '{%s}' sleep 1"
+                             % (",".join(mix_events)), shell=True,
+                             ignore_status=True)
+            er_ln = "WARNING: grouped events cpus do not match, disabling group:"
```

Comment:
it's confusing why an error string change requires.

Reply:
when running mixed group events, warning message changed so updated same in script

`localhost:~ # perf stat -e '{nmem0/cache_rh_cnt/,nmem1/cache_rh_cnt/,nmem2/cache_rh_cnt/}' sleep 1
WARNING: grouped events cpus do not match, disabling group:
  anon group { nmem0/cache_rh_cnt/, nmem1/cache_rh_cnt/, nmem2/cache_rh_cnt/ }

 Performance counter stats for 'system wide':

                 0      nmem0/cache_rh_cnt/                                         
     <not counted>      nmem1/cache_rh_cnt/                                           (0.00%)
     <not counted>      nmem2/cache_rh_cnt/                                           (0.00%)

       1.002028416 seconds time elapsed
`